src/modules/rlm_dotnet/coreclrhost.h

```diff
@@ -0,0 +1,133 @@
+// Retrieved from https://github.com/dotnet/coreclr/blob/master/src/coreclr/hosts/inc/coreclrhost.h
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
This may cause issues with the GPL license.  I'll have to check.

Reply:
OK -- I think I can reproduce the functionality of this file pretty easily if needed.